dashboard/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as path from "path";
+import app from "./app";
+import { db } from "common";
+
+const start = async () => {
+  const PORT = process.env.PORT;
+
+  if (!PORT) {
+    throw new Error("PORT must be defined");
+  }
+
+  process.on("unhandledRejection", (error) => {
+    console.error("unhandledRejection", error);
+  });
```

Comment:
**Issue**: The unhandledRejection listener logs the error but does not exit the process, which might leave the application in an unstable state. <br> **Fix**: It's recommended to exit the process after logging an unhandled rejection to avoid potential instability in the application state. <br> **Code Suggestion**: 
 ```
 +    process.on("unhandledRejection", (error) => {
 +      console.error("unhandledRejection", error);
 +      process.exit(1);
 +    });
 ```